test/integration/test-forms.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "name": "Outlook",
+        "url": "https://outlook.live.com/owa/?authRedirect=true&nlp=1",
+        "expectedFields": ["username", "password"]
+    },
+    {
+        "name": "GitHub",
+        "url": "https://github.com/login",
+        "expectedFields": ["username", "password"]
```

Comment:
Don't you think this might be a little less maintainable though? I mean, if we have 10+ sites in that list, chances are they'll update them at some point. Not sure how necessary it would be to ensure we detect the right ones (usually there aren't more then 1 form on page for login).

If you have any examples of pages that could be potentially recognised as having more than 1 form, it'd be interesting to add those.

Reply:
Yeah, probably true, just throwing ideas out. My first thought was to dump the HTML so that the test cases could run without internet, that would also protect about the updating.

But potentially it's unnecessary since, as you say, "usually there aren't more then 1 form on page for login"

I know that there are a few sites that have signup and login as two different forms next to each other, it would be cool if Locust would be able to detect that 🤔

Reply:
I had a think on your suggestion and I happen to really like the idea.. I've added a new issue for it: #13